cruise-control-client/cruisecontrolclient/client/ExecutionContext.py

```diff
@@ -5,143 +5,93 @@
 import cruisecontrolclient.client.Endpoint as Endpoint
 
 # To be able to make more precise type hints
-from typing import Dict, Tuple, Type
+from typing import Dict, List, Set, Type
 
+# Define the in-order available endpoints for programmatically building the argparse CLI
+AVAILABLE_ENDPOINTS: List[Type[Endpoint.AbstractEndpoint]] = [
+    Endpoint.AddBrokerEndpoint,
+    Endpoint.AdminEndpoint,
+    Endpoint.BootstrapEndpoint,
+    Endpoint.DemoteBrokerEndpoint,
+    Endpoint.FixOfflineReplicasEndpoint,
+    Endpoint.KafkaClusterStateEndpoint,
+    Endpoint.LoadEndpoint,
+    Endpoint.PartitionLoadEndpoint,
+    Endpoint.PauseSamplingEndpoint,
+    Endpoint.ProposalsEndpoint,
+    Endpoint.RebalanceEndpoint,
+    Endpoint.RemoveBrokerEndpoint,
+    Endpoint.ResumeSamplingEndpoint,
+    Endpoint.RightsizeEndpoint,
+    Endpoint.ReviewBoardEndpoint,
+    Endpoint.ReviewEndpoint,
+    Endpoint.StateEndpoint,
+    Endpoint.StopProposalExecutionEndpoint,
+    Endpoint.TrainEndpoint,
+    Endpoint.TopicConfigurationEndpoint,
+    Endpoint.UserTasksEndpoint
+]
 
-class ExecutionContext:
```

Comment:
This is not a requirement, but there isn't a major benefit in storing these properties within a class.  It merely adds an additional layer between the values and things accessing them.